docs/_source/_common/snippets/start_page.md

```diff
@@ -37,15 +36,9 @@ rg.init(
 )
 ```
 
-## 4. Create your first dataset
+# 4. Create your first dataset
 
-1. Specify a workspace where the dataset is to be saved. You can check the available workspaces with the following snippet:
-
-```python
-workspaces = rg.Workspace.list()
-workspace_names = [workspace.name for workspace in workspaces]
-workspace_names
-```
+1. Specify a workspace where the dataset is to be saved. You can check your workspaces in the Argilla UI by clicking on "My settings".
```

Comment:
Maybe add a brief explanation and snippet at the bottom of this section to show how to create a workspace in case they don't have any workspace.

Add a link to the most relevant page for users to read how workspaces work in Argilla

Reply:
but as far as I could check if they do not remove it, it's created at least one by default (admin in HF and argilla in docker), so would this be still needed?

Reply:
that's for the quickstart only? maybe @davidberenstein1957 can confirm, but I think you are right, let's not add more code snippets. I'd suggest adding a sentence like: If you want to create a new workspace, check the docs (check the docs is a link with how to create the workspace with python)

Reply:
Yes I believe normally the workspace for the owner is added by default but I would reference the user/workspace setting as Dani suggestion. And I would use the `rg.set_workspace` command after the `rg.init` because I think it is required now.